modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -174,69 +176,6 @@
             <class name="org.wso2.identity.integration.test.rest.api.server.tenant.management.v1.TenantFailureTest"/>
         </classes>
     </test>
-
-    <test name="is-tests-scim2" preserve-order="true" parallel="false">
```

Comment:
Why these tests are removed? Not intensional?

Reply:
Fixed: 32a8d34b1e5810114348ea59b871025b75632c20